Aktualisierung spdx tags de.md

Blixxxa opened the pull request:
Updated Spdx tags

Blixxxa committed:
Update de.md

Blixxxa committed:
Update nonfree en.md

Blixxxa committed:
Update nonfree de.md

LeChasseur reviewed (changes requested):
Please check "Eclipse Distribution License". SPDX identifier is BSD-3-Clause

Blixxxa committed:
Update de.md

Revisited the Eclipse Distribution License and switched the tag to BSD-3-Clause due to the identical main license text.
